docs/develop/local_dev.mdx

```diff
@@ -0,0 +1,188 @@
+---
+sidebar_position: 2
+description: "Learn how to set up your local dev environment"
+---
+
+import Admonition from '@theme/Admonition';
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+# Local Dev
+
+## Running Restate Server & CLI locally
+<Tabs groupId={"running-restate"}>
+    <TabItem value={"npm"} label={"npm"}>
+        <CH.Spotlight className={"spotlight-medium"}>
+            Install Restate Server and run it:
+            ```shell
+            npm install --global @restatedev/restate-server@latest &&
+            restate-server
+            ```
+        </CH.Spotlight>
+        <CH.Spotlight className={"spotlight-medium"}>
+            Or use npx, without installation:
+            ```shell
+            npx @restatedev/restate-server
+            ```
+        </CH.Spotlight>
+        <CH.Spotlight className={"spotlight-medium"}>
+            Install the Restate CLI via:
+            ```shell
+            npm install --global @restatedev/restate@latest
+            ```
+        </CH.Spotlight>
+
+    </TabItem>
+    <TabItem value={"Homebrew"}  label={"Homebrew"}>
+        <CH.Spotlight className={"spotlight-medium"}>
+
+        Install Restate Server and run it with:
+        ```shell
+        brew install restatedev/tap/restate-server &&
+        restate-server
+        ```
+        </CH.Spotlight>
+
+        <CH.Spotlight className={"spotlight-medium"}>
+        Install the CLI via:
+        ```shell
+        brew install restatedev/tap/restate
+        ```
+        </CH.Spotlight>
+
+    </TabItem>
+    <TabItem value={"bin"} label={"Download binaries"}>
+        Install the Restate Server and CLI by downloading the binaries with `curl` from the [releases page](https://github.com/restatedev/restate/releases/latest), and make them executable:
+
+        <CH.Code rows={"5"}>
+
+            ```shell MacOS-x64
+            curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.x86_64-apple-darwin.tar.gz && \
+            tar -xvf restate.x86_64-apple-darwin.tar.gz && \
+            chmod +x restate restate-server && \
+            sudo mv restate /usr/local/bin/ && \
+            sudo mv restate-server /usr/local/bin/
```

Comment:
Add a `BIN` variable to this and below command, so it's easier to change where you install it. I don't like we have commands with `sudo` in it.

```shell
BIN=/usr/local/bin && \
curl -LO https://github.com/restatedev/restate/releases/latest/download/restate.x86_64-apple-darwin.tar.gz && \
            tar -xvf restate.x86_64-apple-darwin.tar.gz && \
            chmod +x restate restate-server && \
            mv restate $BIN && \
            mv restate-server $BIN
```